source/_components/plant.markdown

```diff
@@ -27,9 +32,42 @@ plant:
     min_temperature: 15
 ```
 
-## Using plain MQTT sensor to get the data
+Configuration variables:
+
+- **simulated_plant** (*Required*):
```

Comment:
This is an entity_id, I did not try but should be anything that also works elsewhere.